### ISSUE TITLE
Demo error on Firefox Developer edition v59.0b4 on Ubuntu 16.04

### ISSUE THREAD

**Issue by Darkilen** [alex3165/react-mapbox-gl#508]
Hello,

I can't see [demos](https://alex3165.github.io/react-mapbox-gl/demos) on Firefox Developer Edition v59.0b4 on Ubuntu 16.04 because i have this error : 
```
The above error occurred in the <Layer> component:
    in Layer
    in EnhancedLayer (created by LondonCycle)
    in div
    in ReactMapboxGl (created by LondonCycle)
    in LondonCycle (created by Demos)
    in div (created by styled.div)
    in styled.div (created by Demos)
    in div (created by styled.div)
    in styled.div (created by Demos)
    in Demos (created by RouterContext)
    in div (created by styled.div)
    in styled.div (created by Root)
    in Root (created by RouterContext)
    in RouterContext (created by Router)
    in Router

Consider adding an error boundary to your tree to customize error handling behavior.
You can learn more about error boundaries at https://fb.me/react-error-boundaries. bundle.js:24001:5
```
```
NS_ERROR_NOT_AVAILABLE:  bundle.js:8796
	getImageData https://alex3165.github.io/react-mapbox-gl/bundle.js:8796:907
	[226]</Map</e.prototype.addImage https://alex3165.github.io/react-mapbox-gl/bundle.js:8786:10225
	Layer/_this.initialize/< https://alex3165.github.io/react-mapbox-gl/bundle.js:41750:21
	forEach self-hosted:271:13
	Layer/_this.initialize https://alex3165.github.io/react-mapbox-gl/bundle.js:41749:17
	Layer.prototype.componentWillMount https://alex3165.github.io/react-mapbox-gl/bundle.js:41768:9
	callComponentWillMount https://alex3165.github.io/react-mapbox-gl/bundle.js:25473:5
	mountClassInstance https://alex3165.github.io/react-mapbox-gl/bundle.js:25530:7
	updateClassComponent https://alex3165.github.io/react-mapbox-gl/bundle.js:25912:9
	beginWork https://alex3165.github.io/react-mapbox-gl/bundle.js:26301:16
	performUnitOfWork https://alex3165.github.io/react-mapbox-gl/bundle.js:28269:16
	workLoop https://alex3165.github.io/react-mapbox-gl/bundle.js:28378:28
	callCallback https://alex3165.github.io/react-mapbox-gl/bundle.js:16995:9
	invokeGuardedCallbackDev https://alex3165.github.io/react-mapbox-gl/bundle.js:17034:7
	invokeGuardedCallback https://alex3165.github.io/react-mapbox-gl/bundle.js:16891:5
	performWork https://alex3165.github.io/react-mapbox-gl/bundle.js:28496:7
	scheduleUpdateImpl https://alex3165.github.io/react-mapbox-gl/bundle.js:28881:19
	scheduleUpdate https://alex3165.github.io/react-mapbox-gl/bundle.js:28820:12
	enqueueSetState https://alex3165.github.io/react-mapbox-gl/bundle.js:25342:7
	ReactComponent.prototype.setState https://alex3165.github.io/react-mapbox-gl/bundle.js:39224:3
	ReactMapboxFactory/_x</ReactMapboxGl.prototype.componentDidMount/< https://alex3165.github.io/react-mapbox-gl/bundle.js:38912:25
	[239]</Evented.prototype.fire https://alex3165.github.io/react-mapbox-gl/bundle.js:8812:833
	[226]</Map</e.prototype._render https://alex3165.github.io/react-mapbox-gl/bundle.js:8786:14876
	<anonyme> self-hosted:992:17
```
All other pages work perfectly.

However, the [demos](https://alex3165.github.io/react-mapbox-gl/demos) work perfectly on Chrome on Ubuntu 16.04

Why is this happening ? 
Thank you in advance

**Comment by redbmk** [comment]
Also looks like it's specific to the London Cycle demo - if you switch to another demo before it finishes loading those work fine. Except the map is huge - maybe there should be a `max-height: 100vh` on the container. Not sure why it does that for Firefox and not Chrome.

**Comment by alex3165** [comment]
Thanks for reporting the issue @Darkilen, I will have a look at this issue as soon as possible. Help is welcome

**Comment by Darkilen** [comment]
Hello, it's fix on firefox 60.0b2 on Ubuntu 16.04, thanks you.
